Dockerfiles/manifests/aws/tf/README.md

```diff
@@ -35,3 +35,16 @@ To allow kubectl to talk to your cluster `make kubeconfig` and follow the sugges
 Make sure nodes can register `make config-map-aws-auth` and wait for nodes to appear.
 
 Your are now ready.
+
+### Setup on an already provisioned Cluster
+
+Get the kubectl command and aws-iam-authenticator command
+Get the kubeconfig file form someone and put it somewhere
+Get the aws credentials and put them in ~/.aws/credentials
```

Comment:
Might be useful to include the steps that is used with the `aws-cli` because I think most people use that. I think you can do a `aws configure --profile {some_name}` if the credentials are not the same as your personal ones.

Reply:
Environment variables work as well - I work with multiple AWS accounts, so I never have credentials file to prevent misuse.  Instead I use `direnv` and change environment on  a fly

Reply:
Same remark as above. I just documented what I did, not sure whether redoing those steps and updating the README is an efficient use of my time